src/PayID/pay-id-client.ts

```diff
@@ -1,5 +1,6 @@
-import axios, { AxiosInstance } from 'axios'
```

Comment:
How do we make HTTP calls now? Is that generated from ApiClient?

Reply:
Yup. The API client codegenerates the API called.